static/app/views/organizationLayout.tsx

```diff
@@ -30,6 +32,7 @@ const OrganizationHeader = HookOrDefault({
  */
 function OrganizationLayout({includeSidebar, children}: OrganizationLayoutProps) {
   const {organization, loading, error, errorType} = useLegacyStore(OrganizationStore);
+  useEnsureOrganization();
```

Comment:
This reads like a void function, can we instead place the effect here and centralize this behavior? Not sure if we need a separate hook and context for this?

Reply:
I was trying to avoid exporting the context basically

Reply:
I think in this case it's better to export the context and directly consume as opposed to a hook. The hook implies a reusable behavior outside of this instance, but it seems like we don't want that and just want to consume the context directly.